src/dotnet/APIView/apiview_token_gist.json

```diff
@@ -0,0 +1,356 @@
+    //     TypeName = 6,
+    //     MemberName = 7,
+    //     StringLiteral = 8,
+    //     Literal = 9,
+    //     Comment = 10,
+    //     DocumentRangeStart = 11,
+    //     DocumentRangeEnd = 12,
+    //     DeprecatedRangeStart = 13,
+    //     DeprecatedRangeEnd = 14,
+    //     SkipDiffRangeStart = 15,
+    //     SkipDiffRangeEnd = 16,
+    //     InheritanceInfoStart = 17,
+    //     InheritanceInfoEnd = 18
+
+    "Name": "ClassLibrary1", // Would be displayed in review list
+    "Tokens": [
+        {
+            "DefinitionId": null, // Allows token to be navigated to, should be unique, used as ID for comment thread
+            "NavigateToId": null, // If set clicking on the token would navigate to other token with this ID
+            "Value": "namespace", // Textual value
+            "Kind": 4 // One of token kinds from above
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1",
+            "NavigateToId": null,
+            "Value": null,
+            "Kind": 5
+        },
+        {
+            "DefinitionId": "ClassLibrary1",
+            "NavigateToId": null,
+            "Value": "ClassLibrary1",
+            "Kind": 0
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "{",
+            "Kind": 3
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": null,
+            "Kind": 1
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "    ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1.Class1",
+            "NavigateToId": null,
+            "Value": null,
+            "Kind": 5
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "public",
+            "Kind": 4
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": "class",
+            "Kind": 4
+        },
+        {
+            "DefinitionId": null,
+            "NavigateToId": null,
+            "Value": " ",
+            "Kind": 2
+        },
+        {
+            "DefinitionId": "ClassLibrary1.Class1",
+            "NavigateToId": null,
+            "Value": "Class1",
+            "Kind": 6,
+            "CrossLanguageDefId": "Namespace1.Class1"
```

Comment:
@lmazuel @weshaggard Here is the new token property added for common ID as per our earlier discussion.